trade-finance-manager-ui/server/helpers/date.js

```diff
@@ -85,7 +85,7 @@ const getIsoMonth = (dateInMonth) => {
   return format(dateInMonth, 'yyyy-MM');
 };
 
-const ISO_MONTH_REGEX = /^\d{4}-\d{2}$/;
+const ISO_MONTH_REGEX = /^(?<year>\d{4})-(?<month>\d{2})$/;
```

Comment:
I know we don't need this change anymore, but I think it is worth including it as it does make the structure of the regex clearer to read. Thoughts?

Reply:
Yep, agreed 👍